packages/secure-store/secure-unlock/walletKeyStore.ts

```diff
@@ -0,0 +1,98 @@
+import { Platform } from 'react-native'
+import * as Keychain from 'react-native-keychain'
+import { type KeychainOptions, getKeychainItemById, storeKeychainItem } from '../keychain'
+
+const walletKeyStoreBaseOptions: KeychainOptions = {
+  /* Only allow the current set of enrolled biometrics to access the wallet key */
+  accessControl: Keychain.ACCESS_CONTROL.BIOMETRY_CURRENT_SET,
+
+  // TODO: might want to use WHEN_UNLOCKED_THIS_DEVICE_ONLY, to allow devices without a passcode (as we have our own passcode and extra biometrics check)
+  // Not sure how it works if you have no passcode, but you do have biometrics (i think that is not possible?)
+  /* Only allow access to the wallet key on the device is was created on, is unlocked, and has a passcode set */
+  accessible: Keychain.ACCESSIBLE.WHEN_PASSCODE_SET_THIS_DEVICE_ONLY,
+
+  // TODO: internationalization
+  authenticationPrompt: {
+    title: 'Unlock wallet',
+    description: 'Access to your wallet is locked behind a biometric verification.',
+  },
+
+  /* Android Only. Ensure wallet key is protected by hardware. Wil results in error if hardware is not available. Hardware is either StrongBox or TEE */
+  securityLevel: Keychain.SECURITY_LEVEL.SECURE_HARDWARE,
+
+  /* Android Only. Ensure wallet key is protected using RSA storage type, this is the only storage type supporting biometrics. */
+  storage: Keychain.STORAGE_TYPE.RSA,
+
+  /* Ensure wallet key is protected by biometrics. It is not possible to fallback to the device passcode if the biometric authentication failed. */
+  authenticationType: Keychain.AUTHENTICATION_TYPE.BIOMETRICS,
+}
+
+const WALLET_KEY_ID = (version: number) => `PARADYM_WALLET_KEY_${version}`
```

Comment:
Paradym specific identifier again.

Reply:
see my previous comment. Any suggestions on what you'd use?

Reply:
We'd likely need a config json for each app which defines the theme stuff but also items like this. Might be a bit overkill though.

Reply:
Yeah I don't really see the added value of that. Then I'd prefer going with `WALLET_KEY_${version}`

Reply:
Okay, paradym prefix is also fine for me. It does not really matter in the end.